### PR TITLE
Configure Node GC Threshold

### LINKED ISSUE #333: Configurable Serf ReconnectTimeout and TombstoneTimeout
Part of our infrastructure running on AWS spot instances. I played with nomad on spot instances and finished with 50 members in "failed" state. I only run 3 instances concurrent, and can imagine what will happened when I will run 50 agents on spot instances. The default 3 days timeout its huge if you running on spot instances.

If you agree I can open PR.

Thanks.

### PR DESCRIPTION
**Pull request by gregory-m**
Close #333

### CODE REVIEW

File: website/source/docs/agent/config.html.md [hunk 1]
```diff
@@ -173,6 +173,8 @@ configured on client nodes.
     sub-schedulers this server will handle. This can be used to restrict the
     evaluations that worker threads will dequeue for processing. This
     defaults to all available schedulers.
+  * `node_gc_threshold` This is a string with a unit suffix, such as "300ms",
+    "1.5h" or "25m". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". Controls how "old" a node must be to be collected by GC
```
Comment (dadgar) [inline website/source/docs/agent/config.html.md]: Would you mind formatting this to 80 characters. I would also reword the last sentence to something like: "Controls how long a node must be in a terminal state before it is garbage collected and purged from the system."

**Comment by dadgar** [comment]
Small comment but looks great!

**Comment by gregory-m** [comment]
Second time I forget about 80 characters will and rule to my linter.

**Comment by dadgar** [comment]
Awesome! Thanks so much!

**Pull request merged**